src/window/window.ts

```diff
@@ -116,7 +116,7 @@ export const StarknetMethodArgumentsSchemas = {
       z
         .object({
           starknetVersion: z
-            .union([z.literal("v3"), z.literal("v4")])
+            .union([z.literal("v3"), z.literal("v4"), z.literal("v5")])
```

Comment:
Why do we need v3 here?

Reply:
kept it for backward compatibility
but with rpc wallet snip and this version it could be probably removed

Reply:
don't we need v6 as well here?

Reply:
this is related to the `enable` function only 
with v6 
```typescript
this.request({
        type: "wallet_requestAccounts",
      })
```

will be used

So probably it would makes sense to remove this part?
@dhruvkelawala @vladutjs